Tests/FluentAssertions.Specs/Primitives/ObjectAssertionSpecs.cs

```diff
@@ -196,6 +196,16 @@ public void Chaining_after_one_assertion()
             // Act / Assert
             value.Should().Be(value).And.NotBeNull();
         }
+
+        [Fact]
+        public void Chaining_after_one_assertion_generic_version()
```

Comment:
🔧 The name doesn't seem proper English. What about `Can_chain_multiple_assertions`?